api/settings/prod.py

```diff
@@ -176,6 +177,64 @@
 EMAIL_HOST_USER = os.environ.get('EMAIL_HOST_USER')
 EMAIL_HOST_PASSWORD = os.environ.get('EMAIL_HOST_PASSWORD')
 
+LOGGING_CONFIG = None
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'verbose': {
+            'format': '%(levelname)s %(asctime)s %(module)s %(process)d ' +
+            '%(thread)d %(message)s',
+        },
+        'simple': {
+            'format': '%(levelname)s %(message)s'
+        },
+        'basic': {
+            'format': '[%(levelname)s] %(name)s: %(message)s'
+        },
+    },
+    'filters': {
+        'require_debug_false': {
+            '()': 'django.utils.log.RequireDebugFalse'
+        }
+    },
+    'handlers': {
+        'null': {
+            'level': 'DEBUG',
+            'class': 'logging.NullHandler',
+        },
+        'debug_file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': '/var/log/hs-api/debug.log',
```

Comment:
Create a variable `LOG_FILE_PATH`
* It breaks because there is no `/var/log/hs-api/` directory. We need to create the directory while project setup. 
* Is there a way to create the file automatically? we can have this file in `~/.logs/hs-api/` to tackle root permission issues.
```
FileNotFoundError: [Errno 2] No such file or directory: '/var/log/hs-api/debug.log'
```

Reply:
In dev/prod, the created user (non-root) will have sudo access. So, I don't think there should be any concern about root permissions.

3 commands need to be run for this with sudo access in setup phase:
```
mkdir /var/log/hs-api
touch /var/log/hs-api/debug.log
chown -R <user>:<user> /var/log/hs-api
```